base/src/com/google/idea/blaze/base/buildview/BazelExecService.kt

```diff
@@ -97,6 +97,7 @@ class BazelExecService(private val project: Project) : Disposable {
       handler.addProcessListener(object : ProcessListener {
         override fun onTextAvailable(event: ProcessEvent, outputType: Key<*>) {
           ctx.output(PrintOutput.process(event.text))
+          LOG.debug("BUILD OUTPUT: " + event.text)
```

Comment:
clean this up?

Reply:
No I would suggest to keep this. Because this way it is pretty easy to get the build log in integration tests: `cat <log file> | grep "BUILD OUTPUT"`